src/sgx/signing/signing_public.h

```diff
@@ -0,0 +1,36 @@
+#ifndef SIGNING_PUBLIC_H_
```

Comment:
nit: `#pragma once` is supported by all the compilers that matter these days

Reply:
👍